### ISSUE TITLE
Competition dataset not linking with Colab

### ISSUE THREAD

**Issue by mihirdcoder** [Kaggle/kaggle-api#99]
I am trying to use colab API to work for nyc taxi fare competition. But it is giving me error while fetching datasets.

I have uploded kaggle API key, placed it in root folder but when I try to run
!kaggle competitions download -c new-york-city-taxi-fare-prediction

It is throwing 404 error !

**Comment by mihirdcoder** [comment]
Yes I am able to download, I an able to use data grom gdrive but can't connect it directly from kaggle API, the !kaggle datasets list command also does not work, throws same error.

**Comment by corrieann** [comment]
Can you share your Colab notebook?
It's working for me --
https://colab.research.google.com/gist/corrieann/3abf2e79f833200d72b58ff10b1df39e/kaggle-api.ipynb

**Comment by jzuern** [comment]
@corrieann Is the notebook still working for you? I tried to run it and it throws a 404 when trying to list the datasets. Might be a kaggle server issue?

**Comment by corrieann** [comment]
Still works for me.

Maybe try https://colab.research.google.com/github/corrieann/kaggle/blob/master/kaggle_api_in_colab.ipynb
It avoids using Google Drive for fetching your kaggle.json file which is, I suspect, where you're running into trouble.
